jenkins_job_manager/cli.py

```diff
@@ -174,7 +187,7 @@ def jjm_apply(obj: JenkinsJobManager, target: str, allow_delete: bool):
     if obj.detected_changes() is False:
         click.secho("No changes to apply.", fg="green")
         return
-    handle_plan_report(obj, use_pager=False)
+    handle_plan_report(obj, use_pager=False, report_format="default")
```

Comment:
if the default is `"default"`, we can set that in the function signature where  `handle_plan_report` is defined.